src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -204,29 +164,6 @@ public function testExpandedChoicesOptionsAreFlattened()
         $form = $this->factory->create('Symfony\Component\Form\Extension\Core\Type\ChoiceType', null, array(
             'expanded' => true,
             'choices' => $this->groupedChoices,
-            'choices_as_values' => true,
-        ));
-
-        $flattened = array();
-        foreach ($this->groupedChoices as $choices) {
-            $flattened = array_merge($flattened, array_keys($choices));
-        }
-
-        $this->assertCount($form->count(), $flattened, 'Each nested choice should become a new field, not the groups');
-
-        foreach ($flattened as $value => $choice) {
-            $this->assertTrue($form->has($value), 'Flattened choice is named after it\'s value');
-        }
-    }
-
-    /**
-     * @group legacy
-     */
-    public function testExpandedChoicesFlippedOptionsAreFlattened()
```

Comment:
yes, the diff is misleading but really only legacy tests are and should be removed here